src/patternfly/components/Drawer/drawer.scss

```
@@ -97,6 +98,12 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   @media screen and (min-width: $pf-global--breakpoint--md) {
     min-width: var(--pf-c-drawer__panel--MinWidth);
   }
+
+  &.pf-m-inline {
```

I don't think you need to nest this selector

```suggestion
  &.pf-m-inline .pf-c-drawer__panel:not(.pf-m-no-border) {
```